index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+class Database {
+    private $conn;
+
+    public function __construct() {
+        $this->conn = new mysqli("localhost", "root", "password", "my_database");
+
+        if ($this->conn->connect_error) {
```

Comment:
Use exceptions for error handling:
if ($this->conn->connect_error) {    
    throw new Exception("Connection failed: " . $this->conn->connect_error);
}